lib/util.js

```diff
@@ -199,17 +199,40 @@ function pad(n) {
 }
 
 /**
- * @param {string} format
+ * @param {string} code
+ * @returns {string}
+ */
+function escapeStyleCode(code) {
+  return `\u001b[${code}m`;
+}
+
+/**
+ * @param {string | string[]} format
  * @param {string} text
  * @returns {string}
  */
 function styleText(format, text) {
   validateString(text, 'text');
+  if (ArrayIsArray(format)) {
+    let left = '';
+    let right = '';
+    for (const key of format) {
+      const formatCodes = inspect.colors[key];
+      if (formatCodes == null) {
+        validateOneOf(key, 'format', ObjectKeys(inspect.colors));
+      }
+      left += escapeStyleCode(formatCodes[0]);
+      right = `${escapeStyleCode(formatCodes[1])}${right}`;
+    }
+
+    return `${left}${text}${right}`;
+  }
+
```

Comment:
I'm fine with merging the PR with the current state, but:

We need a `validateString(format, 'format')` call in here. Previous implementation was missing it as well, and assumes that `format` is a string.

Reply:
If it wasnt it wouldnt satisfy
```js
const formatCodes = inspect.colors[format];
```

Reply:
> If it wasnt it wouldnt satisfy

Not so; consider the value `{ toString(){ 'red'; } }`. That would not pass `validateString(format, 'format')` but will pass `inspect.colors[format]`.